profiles/seccomp/seccomp.go

```diff
@@ -95,6 +96,21 @@ func setupSeccomp(config *types.Seccomp, rs *specs.Spec) (*specs.LinuxSeccomp, e
 
 	newConfig.DefaultAction = specs.LinuxSeccompAction(config.DefaultAction)
 
+	var currentKernelVersion *kernel.VersionInfo
+	kernelGreaterEqualThan := func(v string) (bool, error) {
```

Comment:
not sure I like defining local closures like this, can you just make it a helper function external to this? (yes, Go should allow local functions...)